src/Middleware/tests/cardconnect.tests/CardConnectTests.cs

```diff
@@ -40,6 +45,20 @@ public void to_credit_card_display_test()
             Assert.AreEqual("8025", amex);
         }
 
+        [Test]
+        public void verify_mock_responses_Auth()
+        {
+            var response = _service_no_config.AuthWithCapture(new CardConnectAuthorizationRequest() { amount= "10000" });
```

Comment:
nitpick but spacing on `amount= "10000"` seems off. Mind autoformatting this file?